pkg/resources/database/resource_kafka_test.go

```diff
@@ -158,7 +245,7 @@ func CheckExistsKafka(name string, data *TemplateModelKafka) error {
 
 	res, err := client.GetDbaasServiceKafkaWithResponse(ctx, oapi.DbaasServiceName(name))
 	if err != nil {
-		return err
+		return fmt.Errorf("aww")
```

Comment:
```suggestion
		return err
```
leftover from debugging? :)